app/components/UserValidator/index.tsx

```diff
@@ -81,36 +144,38 @@ const Validator = (props: Props) => {
           }
         });
     },
-    [clearSearch, handleSelect, validateAbakusGroup]
+    [clearSearch, displayResult, handleSelect, validateAbakusGroup]
   );
 
   const handleScannerResult = (scannerResult: string) => {
-    if (scannerResult.length > 0 && !completed) {
-      onSelect({
-        username: scannerResult,
-      });
+    if (scannerResult.length > 0 && !isLoading && !successMessage) {
+      onSelect({ username: scannerResult });
     }
   };
 
   return (
     <>
-      <div
-        className={cx(styles.overlay, {
-          [styles.shown]: completed,
-        })}
+      <Modal
+        show={successMessage !== null}
+        onHide={() => setSuccessMessage(null)}
       >
-        <h3>
-          Tusen takk! Kos deg{' '}
-          <span role="img" aria-label="smile">
-            😀
-          </span>
-        </h3>
-        <Icon name="checkmark" success size={160} />
-      </div>
+        <Flex
+          alignItems="center"
+          column
+          justifyContent="center"
+          padding={'2rem 0'}
+        >
+          <h3>{successMessage}</h3>
+          <Icon name="checkmark" success size={160} />
```

Comment:
By the way, the `success` here does not do anything. I see why though, so just keep it and I will fix it shortly